zds/pages/urls.py

```diff
@@ -16,7 +16,7 @@
 
 urlpatterns = [
     # single pages
-    path("apropos/", about, name="pages-about"),
+    path("apropos/", about, name="pages-technologies"),
```

Comment:
```suggestion
    path("technologies/", about, name="pages-technologies"),
```
Pour être cohérent. Ça ne pose pas de problème, je dirais qu'on s'en fout du référencement pour cette page.

Reply:
C'est fait.